blocks/server-side/hero/hero.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+
+import cn from 'classnames';
+
+import styles from './hero.module.css';
+import { useTextStyles } from '@rescui/typography';
+
+import '@jetbrains/kotlin-web-site-ui/out/components/layout';
+
+import { Button } from '@rescui/button';
+
+import Img from 'next/image';
+import Link from 'next/link';
+
+import heroImg from './images/hero-graphics.webp'
+
+export const ServerSideHero: FC = ({}) => {
+
+    const textCn = useTextStyles();
+
+    return (
+        <div className={styles.wrapper}>
+
+            <Img className={styles.heroImage} src={heroImg} alt={'Server-side graphic'} width={1000} height={1000} />
+
+            <div className={cn('ktl-layout', 'ktl-layout--center')}>
+                <div className={styles.content}>
+                    <h1 className={cn(textCn('rs-hero'), styles.heroTitle)}>
+                        Modern backend<br /> development <br /> with Kotlin
+                    </h1>
+
+                    <p className={cn(textCn('rs-subtitle-2'), styles.heroSubtitle)}>
+                        Kotlin is a perfect fit for building backends on JVM. It's trusted by major tech companies to build scalable and reliable server-side applications
+                    </p>
+
+                    <div className={styles.featuresList}>
+
+                        <Link href={'#safety'} className={styles.feature}>
+                            <p className={cn(textCn('rs-subtitle-2'), styles.featureTitle)}>Safety</p>
```

Comment:
the P tag seems to be redundant here.  The typography class can be applied to the ancestor